variants/tbeam_TFT/platformio.ini

```diff
@@ -0,0 +1,14 @@
+; The 1.0 release of the TBEAM board 
+[env:tbeam_TFT]
```

Comment:
Please rename the variant to something else. the tft suffix is only meant for devices running MUI and is parsed in some install scripts.  My suggestion would be tbeam-displayshield

Reply:
Also, why not derive it from the base tbeam definition in the same environment, like we do for instance in t-deck?